src/Common/src/Interop/Windows/mincore/Interop.GetPriorityClass.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Win32.SafeHandles;
+using System;
+using System.Runtime.InteropServices;
+using System.Security;
```

Comment:
You might want to double check that all of these imported namespace are necessary in these interop files.  Not a big deal, but it looks like a bunch might be unnecessary.